gen/src/write.rs

```diff
@@ -1205,9 +1205,19 @@ fn write_type(out: &mut OutFile, ty: &Type) {
         Type::SliceRef(slice) => {
             write!(out, "::rust::Slice<");
             if slice.mutability.is_none() {
-                write!(out, "const ");
+                match slice.inner {
+                    Type::Ref(_) => {
```

Comment:
@dtolnay Do I need to consider other types here?

Reply:
Yeah slices of pointers are still affected:

```console
$ echo '#[cxx::bridge] mod ffi { extern "Rust" { fn f(x: &[*const u8]); }}' | cargo run --manifest-path gen/cmd/Cargo.toml -- --header - | tail -1

void f(::rust::Slice<const const ::std::uint8_t *> x) noexcept;
```

Reply:
> @dtolnay Do I need to consider other types here?

You